src/PolyhedralGeometry/Visualization.jl

```diff
@@ -1,7 +1,12 @@
 @doc Markdown.doc"""
     visualize(P::Polyhedron)
 
-Visualize a polyhedron.
+Visualize a polyhedron of dimension at most four.
+In dimensions up to 3 a usual embedding is shown.
```

Comment:
in 3 space.

Reply:
Done: https://github.com/oscar-system/Oscar.jl/pull/1911/commits/8b3861f8608b33f26e35bd91090d69cb8ac0c0db